include/galsim/SBAiryImpl.h

```diff
@@ -96,7 +95,7 @@ namespace galsim {
     class AiryInfoObs : public AiryInfo
     {
     public:
-        AiryInfoObs(double obscuration, const GSParamsPtr& _gsparams);
+        AiryInfoObs(double obscuration, GSParamsPtr _gsparams);
```

Comment:
No good reason.  I think it's a remnant of some code changes I was trying out where I got rid of GSParamsPtr entirely and switched to copying the GSParams object into the C++ layer, but then I decided I wanted to keep it this way, so I put them back in and didn't use the same syntax.

So I can switch it back.  The const at least is probably worth keeping as a contract that this won't change the gsparam values.  The reference isn't really that important when passing a smart pointer though, since one point of the smart pointer is to make it safe to pass by value.